chain/client/src/sync/state.rs

```diff
@@ -1466,6 +1504,46 @@ pub fn external_storage_location(
     )
 }
 
+/// multi node dump: Construct a location on the external storage.
+pub fn external_storage_location_multi_node(
+    chain_id: &str,
+    epoch_id: &EpochId,
+    epoch_height: u64,
+    shard_id: u64,
+    part_id: u64,
+    num_parts: u64,
+) -> String {
+    format!(
+        "multi_node_tmp_1/{}/{}",
```

Comment:
Why `tmp`?

Reply:
will remove, this was handy for testing different setups